sympy/plotting/experimental_lambdify.py

```diff
@@ -187,8 +187,9 @@ def __init__(self, args, expr):
                                                  use_python_cmath=True)
         self.failure = False
 
-    def __call__(self, args):
-        args = complex(args)
+    def __call__(self, args, kwargs = {}):
+        if kwargs.get('complex_wrap_evalf', None) is not None:
```

Comment:
Skip `is not None`?

Reply:
Or stricter `if kwargs.get('complex_wrap_evalf', None) is not False:`. In this case if it is not specified, it will be attempted.

Reply:
@smichr ```if kwargs.get('complex_wrap_evalf', None) is not False:``` This is failing to pass. I think it is more strict for other examples and this one too.

Reply:
@oscargus I write it just to make it more clear for other contributors and to mae it more robust. Your way is also right.

Reply:
@jmig5776 I agree. Maybe these should be some guidelines what is preferred as I've seen both options.

Maybe (just maybe) @smichr meant `if kwargs.get('complex_wrap_evalf', False) is not False:`?